tyssue/draw/plt_draw.py

```diff
@@ -119,57 +119,111 @@ def create_gif(
         shutil.rmtree(graph_dir)
 
 
-def sheet_view(sheet, coords=COORDS, ax=None, **draw_specs_kw):
-    """Base view function, parametrizable
+def sheet_view(sheet, coords=COORDS, ax1=None, ax2=None, **draw_specs_kw):
+    """ Base view function, parametrizable
     through draw_secs
-
     The default sheet_spec specification is:
-
-    {'edge': {
-      'visible': True,
-      'width': 0.5,
-      'head_width': 0.2, # arrow head width for the edges
-      'length_includes_head': True, # see matplotlib Arrow artist doc
-      'shape': 'right',
-      'color': '#2b5d0a', # can be an array
-      'alpha': 0.8,
-      'zorder': 1,
-      'colormap': 'viridis'},
-     'vert': {
-      'visible': True,
-      's': 100,
-      'color': '#000a4b',
-      'alpha': 0.3,
-      'zorder': 2},
-     'face': {
-      'visible': False,
-      'color': '#8aa678',
-      'alpha': 1.0,
-      'zorder': -1}
-      }
+    {"edge": {
+	"visible": true,
+    "width": 0.5,
+    "head_width": 0.0,
+    "length_includes_head": true,
+    "shape": "right",
+    "color": "#2b5d0a",
+    "alpha": 0.8,
+    "zorder": 1,
+	"colormap": "viridis"
+    },
+    "vert": {
+    "visible": false,
+    "s": 100,
+    "color": "#000a4b",
+    "alpha": 0.3,
+    "zorder": 2
+    },
+    "grad": {
+    "color":"#000a4b",
+    "alpha":0.5,
+    "width":0.04
+    },
+    "face": {
+    "visible": false,
+    "color":"#8aa678",
+    "alpha": 1.0,
+    "zorder": -1
+    },
+    "axis": {
+    "autoscale": true,
+    "color_bar": false,
+    "color_bar_cmap":"viridis",
+    "color_bar_range":false, 
+    "color_bar_label":false, 
+    "color_bar_target":"face"}
+    }
+    Important note for quantitative colormap plots: make sure to normalize your values before getting
+    the colors using draw_specs["face"]["color"] = cmap(pandas_holding_quantity_of_interest).
+    For each plot normalize with respect to the current values (max and min) such that they lie between and including 0 to 1.
+    Note that if you want to keep a constant colorbar range you have to choose the normalization to match
+    the max and min of the color bar range you chose. 
     """
     draw_specs = sheet_spec()
     spec_updater(draw_specs, draw_specs_kw)
-    if ax is None:
-        fig, ax = plt.subplots()
+
+    if (ax1 is None) or (ax2 is None):
+        fig = plt.figure()
     else:
-        fig = ax.get_figure()
+        fig = plt.get_figure()
```

Comment:
I'm not a big fan of `plt.get_figure` because it's state dependant (it's the current active figure), `ax.get_figure` is more robust